docs/content/installation/metrics/metrics-configuration.md

```diff
@@ -16,6 +16,12 @@ These variables affect the general configuration of PostgreSQL Operator Monitori
 
 | Name | Default | Required | Description |
 |------|---------|----------|-------------|
+| `alertmanager_log_level` | info |  | Set the log level for Alertmanager logging. |
+| `alertmanager_service_type` | ClusterIP | **Required** | The service type to user for the Alertmanager service. |
```

Comment:
`user` appears to be a typo.

I can't think of a succinct way to explain "service type" so my inclination is to use Kubernetes language and link to their docs. You can take it or leave it!

```suggestion
| `alertmanager_service_type` | ClusterIP | **Required** | How to [expose][k8s-service-type] the Alertmanager service. |
```

```markdown
[k8s-service-type]: https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types
```